code/modules/mob/living/simple_animal/abnormality/aleph/melting_love.dm

```diff
@@ -156,65 +153,43 @@
 	base_pixel_x = -32
 	desc = "A pink hunched creature with long arms, there are also visible bones coming from insides of the slime."
 	if(istype(gifted_human))
-		to_chat(gifted_human, span_userdanger("You feel like you are about to burst !"))
 		gifted_human.emote("scream")
-		if(KillGifted())
-			UnregisterSignal(gifted_human, COMSIG_LIVING_DEATH)
-			UnregisterSignal(gifted_human, COMSIG_WORK_COMPLETED)
+		gifted_human.death()
```

Comment:
This removes the ability to survive losing the gift from ML which was introduced in one of the more recent reworks.